dts/bindings/led/pwm-leds.yaml

```diff
@@ -5,14 +5,27 @@ description: PWM LEDs parent node
 
 compatible: "pwm-leds"
 
+properties:
+    label:
+      required: false
+      type: string
+      description: |
+        Human readable string describing the device and used to set the device
+        name. It can be passed as argument to device_get_binding() to retrieve
+        the device. If this property is omitted, then the device name is set
+        from the node full name.
+
 child-binding:
     description: PWM LED child node
     properties:
         pwms:
-          type: phandle-array
           required: true
+          type: phandle-array
```

Comment:
Extraneous diff; please remove.

Reply:
This one was intentional although not mentioned in the commit message. This allows to have the same ordering in the file. If OK, I'd like to keep it.